lib/ps_set.fz

```diff
@@ -60,7 +60,7 @@ is
 
   # add new element k to this set.
   #
-  add (k K) ref ps_set K is  # NYI: #167: 'ref' due to lack of 'like this'
+  add (k K) ref ps_set K is
```

Comment:
The idea was to have 
```
  add (k K) ps_set.this.type is
```
but then the result in the `else` branch is incompatible. I am not sure if making this `fixed` works, but it probably should:
```
  fixed add (k K) ps_set.this.type is
```
This would prevent inheriting `add`, i.e., in an heir feature, `add` would be abstract unless explicitly implemented.